core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -61,17 +61,17 @@ import scala.util.control.ControlThrowable
  *  - data-plane :
  *    - Handles requests from clients and other brokers in the cluster.
  *    - The threading model is
- *      1 Acceptor thread per listener, that handles new connections.
- *      It is possible to configure multiple data-planes by specifying multiple "," separated endpoints for "listeners" in KafkaConfig.
- *      Acceptor has N Processor threads that each have their own selector and read requests from sockets
- *      M Handler threads that handle requests and produce responses back to the processor threads for writing.
+ * 1 Acceptor thread per listener, that handles new connections.
+ * It is possible to configure multiple data-planes by specifying multiple "," separated endpoints for "listeners" in KafkaConfig.
+ * Acceptor has N Processor threads that each have their own selector and read requests from sockets
+ * M Handler threads that handle requests and produce responses back to the processor threads for writing.
  *  - control-plane :
  *    - Handles requests from controller. This is optional and can be configured by specifying "control.plane.listener.name".
- *      If not configured, the controller requests are handled by the data-plane.
+ * If not configured, the controller requests are handled by the data-plane.
  *    - The threading model is
- *      1 Acceptor thread that handles new connections
- *      Acceptor has 1 Processor thread that has its own selector and read requests from the socket.
- *      1 Handler thread that handles requests and produce responses back to the processor thread for writing.
```

Comment:
I guess this change is completed by fomatter tool? You might need to manually revert these changes, and some empty line changes that doesn't relate to your fix.